tests/llmobs/test_llmobs_decorators.py

```diff
@@ -536,3 +536,121 @@ def f(prompt, arg_2, kwarg_1=None, kwarg_2=None):
                 output_value="my custom output",
             )
         )
+
```

Comment:
should we add some test cases to ensure decorator generators are still compatible with `send`, `throw`..?

Reply:
good callout, let me try it out manually and then add a test if needed!